app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -168,15 +169,19 @@ enum class NavigationItem(
 
     OtherUserProfile(
         primaryRoute = OTHER_USER_PROFILE,
-        canonicalRoute = "$OTHER_USER_PROFILE/{$EXTRA_USER_DOMAIN}/{$EXTRA_USER_ID}",
+        canonicalRoute = "$OTHER_USER_PROFILE?$EXTRA_USER_ID={$EXTRA_USER_ID}",
```

Comment:
I'm just curious - is it required for the route to have format `something?userId=userIdValue` instead of just `something/userIdValue`? I find the second one simpler that's why I'm asking if there's any issue with such format? 🤔

Reply:
yes, only `something?userId=userIdValue` works properly with deeplinks